src/contributor-guide/dod/backward-compatibility.md

```diff
@@ -0,0 +1,36 @@
+---
+group: dod
+title: DoD - Backward Compatibility
+---
+
+## Overview
+
+All backward incompatible changes should be approved and covered by static tests to meet the "Definition of Done" (DoD).
```

Comment:
What does `and covered by static tests` mean in this context? Does it mean new static test, exclusion of the code bunch or referencing results of existing tests?

Reply:
Static tests should fail for the code that is going to be broken by the change.
For example, if a method was removed - a static test should be added checking for calls to the removed method.